irohad/model/execution/command_executor.hpp

```diff
@@ -110,6 +112,9 @@ namespace iroha {
 
       bool isValid(const Command &command,
                    ametsuchi::WsvQuery &queries) override;
+
+     private:
+      Account creator_;
```

Comment:
This is not used anywhere. Consider removing

Reply:
It is used  in isValid method